io.github.philippkosarev.bmi.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id" : "io.github.philippkosarev.bmi",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "bmi",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "bmi",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/PhilippKosarev/bmi"
```

Comment:
a commit and/or tag is required.

Reply:
do you mean a `sha256:` line?

Reply:
No https://docs.flatpak.org/en/latest/module-sources.html#git-sources